content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -160,6 +157,7 @@ metadata:
   namespace: bar
   annotations:
     auth.istio.io/8000: NONE
+---
```

Comment:
Do we do this on our other samples?

Reply:
nah - we should for consistency - but I can revert this and others.